Marlin/Configuration_adv.h

```diff
@@ -1594,6 +1594,10 @@
 
   //#define SOUND_MENU_ITEM   // Add a mute option to the LCD menu
   #define SOUND_ON_DEFAULT    // Buzzer/speaker default enabled state
+  
+  #ifdef U8GLIB_SSD1309
```

Comment:
This should be `#if ENABLED(U8GLIB_SSD1309)`

Reply:
done